content/en/_index.md

```diff
@@ -1,11 +1,25 @@
 ---
 date: 2023-02-09T10:21:41-06:00
-headertext: Secure data collaboration,
+headertext: Real-time data collaboration,
 headertextmore: no matter where your data lives.
 headeractions:
 - action: Create an Account
   link: "https://rotational.app/register"
 - action: Learn more
   link: "/ensign/"
-image: "img/horizontal-global-data-flow.png"
 ---
+
+{{< home-intro image="img/global_data_blue_flow.png" >}}
+**Ensign for Data Teams: Real-time Insights**
+
+Ensign is a cloud native, cloud agnostic, real-time data streaming platform built for
+modern data teams.
+
+- Set up fast, secure, and automated data streams for rapid prototyping
+  and faster insights
+- Streamline ETL and MLOps
+- No specialized skills, infra, or surprise costs
+
+Get started in minutes with an API key and developer-friendly SDKs. Made to fit any
+tech stack.
+{{< /home-intro >}}
```

Comment:
Rather than using partials in the layouts (e.g. editing index.html) I'm using shortcodes so that it's easier for us to edit the content in the markdown files.